client/my-sites/posts/post-controls.jsx

```diff
@@ -64,7 +64,7 @@ const getAvailableControls = props => {
 
 		controls.main.push( {
 			className: 'stats',
-			href: `/stats/post/${ post.ID }/${ site.slug }`,
+			href: `/stats/post/${ post.ID }/${ siteSlug }`,
```

Comment:
Makes sense, but what can we do to resolve it? Is `QuerySites siteId={ post.site_ID }` in every post a possible solution? Because these appear to never load, even after some time 😕